blog-tech/2022-10-18-shifu-cloud.md

```diff
@@ -1,5 +1,209 @@
-# shifu.cloud Meetup
+# Shifu Cloud Debut
 
-:::caution
-This blog only supports Chinese version currently...
-:::
+On 2022.10.18, [Shifu Cloud](https://shifu.cloud) was officially released to the public, and users can quickly integrate their devices into ***Shifu*** by visually configure the devices they need to access.
```

Comment:
by visually configure the devices they need to access. => simply filling in fields in the UI